lib/logging/GhostLogger.js

```diff
@@ -239,6 +244,7 @@ class GhostLogger {
                     originalUrl: req.originalUrl,
                     params: req.params,
                     headers: this.removeSensitiveData(req.headers),
+                    body: reqBody,
```

Comment:
I guess that's a bit nicer @gargol. What would be the difference between that and say:
```
req: (req) => {
                var formatted = {
                    meta: {
                        requestId: req.requestId,
                        userId: req.userId
                    },
                    url: req.url,
                    method: req.method,
                    originalUrl: req.originalUrl,
                    params: req.params,
                    headers: this.removeSensitiveData(req.headers),
                    query: this.removeSensitiveData(req.query)
                };

                if (this.logBody) {
                  formatted.body = this.removeSensitiveData(req.body);
                }

                return formatted;
            },
```

Is there a reason to use `const` (if the value can change afterwards)? Is there a reason for using `Object.assign...` over `formatted.body = `.  Asking because I genuinely don't know. Maybe one is more efficient than the other or something?

Reply:
This made `var` / `let` / `const` a lot clearer: https://tylermcginnis.com/var-let-const/ we seem to use them interchangeably throughout various bits of source code.

Reply:
@pysysops regarding a difference between:
```
 formatted.body = this.removeSensitiveData(req.body);
```
and
```
Object.assign(formatted, {
    body: this.removeSensitiveData(req.body)
});
```
There's no specific reason or benefit int this case. I just find having Object.assign more flexible when extending objects (you can pass multiple objects into it and it merges properties in order). But in this case, it might bring more confusion than actual benefit so would go with the variant you suggested :) 

Regarding var/let/const usages. We have a mix in our codebases, that's true. In Ghost server, we are trying to follow the so-called "constantly const" rule (official node.js docs use constantly const as well). In Admin-Client the preference is to "liberal let". It's a bit of a flamewar topic to be honest :smile: Personally use `const` everywhere unless it's specifically needed to use `let`.

Reply:
Thanks for the explanation(s) @gargol 😊 another small gap filled in my knowledge.